r/vignettes/dataset.Rmd

```diff
@@ -277,6 +283,8 @@ avoid scanning because they have no rows where `total_amount > 100`.
 
 ## More dataset options
 
+<!-- ADD SOMETHING HERE ABOUT HOW YOU CAN PASS A SINGLE FILE PATH OR VECTOR OF FILE PATHS TO open_dataset() -->
```

Comment:
Yes, particularly as it relates to write_dataset(). The key point there is that you could have a CSV file that is too big to read into memory, and you could partition it into a bunch of parquet files that are more manageable in size without having to find some way to first read the file into R.